anet-dictionary.yml

```diff
@@ -473,7 +473,7 @@ dictionary:
               visibleWhen: $[?(@.colourOptions === 'GREEN')]
 
     location:
-      format: LAT_LON
```

Comment:
There is still a need for selecting a format for ANET. I suggest to keep the `format` field name